### PR TITLE
Project Challenge

### PR DESCRIPTION
**Pull request by akaTwiiX**
# Description
I build a Docker image with my Dockerfile. Build a postgres container with the Docker Hub postgres image and my Docker container.
I changed code fails and add my README.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+RUN apt-get update && \
+    apt-get install -y build-essential gcc netcat-openbsd
+
+# Install requirements
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt
+
+RUN chmod +x ./entrypoint.sh
+
```
Comment (spmse) [inline Dockerfile:16]: add `EXPOSE <PORT>`

File: README.md [hunk 2]
```diff
@@ -97,6 +97,85 @@ The behavior of some of the views had to be modified to address functionalities
...
+    RUN chmod +x ./entrypoint.sh
+
+    ENTRYPOINT ["/app/entrypoint.sh"]
+    ```
+
+1. create a docker image
+
+    ```bash
+    docker build -t <image-name>:<tag-name> .
+    # -t Name and optionally a tag in the <image-name>:<image-tag> format
```
Comment (spmse) [inline README.md:130]: add real build instruction + name you used for better traceability

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+COPY . $WORKDIR
+
+RUN apt-get update && \
+    apt-get install -y build-essential gcc netcat-openbsd
+
+# Install requirements
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt
+
+RUN chmod +x ./entrypoint.sh
```
Comment (spmse) [inline Dockerfile:15]: add to previous layer

**Review by spmse — CHANGES_REQUESTED** [review]
Hi John, already looks good to me. Adjust the mentioned things and we're good to merge!

File: README.md [hunk 4]
```diff
@@ -128,6 +130,11 @@ The behavior of some of the views had to be modified to address functionalities
     # -t Name and optionally a tag in the <image-name>:<image-tag> format
     ```
 
+    ```bash
+    # e.g.
+    docker build -t trucksign:0.0.0.1 .
```
Comment (spmse) [inline README.md:135]: why the 4th digit? semantic versions are usually split into MAJOR.MINOR.PATCH versions which is why its 3 digits in 1.0.0

just randomly asking

**Review by spmse — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (akaTwiiX) [reply README.md:135]: I didn't even know that there was a standard xDDD

**Pull request merged**